src/app/drivingLicence/fieldsHelper.js

```diff
@@ -6,10 +6,16 @@ module.exports = {
     _value,
     length,
     firstNameField,
-    middleNameField
+    middleNameField,
+    licenceIssuerField
   ) {
     const firstName = this.values[firstNameField];
     const middleName = this.values[middleNameField];
+    const issuer = this.values[licenceIssuerField];
+
+    if (issuer === "DVA"){
```

Comment:
Will we not need two lengths the ticket states first name 18 and second name 18. The following would allow a first name and middle name combination less than 36 but would allow say 20 and a 16 which would be invalid downstream

Reply:
So I was under the impression that DVAs API validates the first/middle name fields together as one string, in the same way as DCS...which would mean we would still want a user to submit a request if their first name is 20 and middle name is 16, will need to confirm as you're right if they validate the 2 fields separately then this would fail DVAs validation